roles/lib_dyn/tasks/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+# tasks file for lib_dyn
```

Comment:
Is the idea to just use the role for making the module available? Seems like it could be nice to pass in a list of records to create, zone and required auth parameters to the role and have the role iterate through the list and create and/or delete the associated entries as needed.